src/legacy/core_plugins/vis_type_xy/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { resolve } from 'path';
+import { Legacy } from 'kibana';
+
+import { LegacyPluginApi, LegacyPluginInitializer } from '../../types';
+
+export interface ConfigShema {
+  visTypeXy: {
+    enabled: boolean;
+  };
+}
+
+const visTypeXyPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPluginApi) =>
+  new Plugin({
+    id: 'vis_type_xy',
```

Comment:
Should we use camelCase id here too directly, since we might need to once we're moving to the new platform?

Reply:
Changed in 4ff43c5. It looks like it shows a JSON schema warning using camelCase but runs fine.

![image](https://user-images.githubusercontent.com/19007109/73665194-4f580b00-4666-11ea-836f-e0217033c210.png)